### ISSUE TITLE
Improve identifier scheme guessing algorithm + add tests

### ISSUE THREAD

**Issue by amoeba** [ec-geolink/d1lod#25]
Just found this in the export:

``` ttl
       glbase:hasIdentifier [
            glbase:hasIdentifierScheme datacite:uri ;
            glbase:hasIdentifierValue "http://dx.doi.org/10.5061/dryad.11k3m?ver=2015-12-02T13:05:03.510-05:00" ;
            a glbase:Identifier ;
            rdfs:label "http://dx.doi.org/10.5061/dryad.11k3m?ver=2015-12-02T13:05:03.510-05:00"
```

**Comment by amoeba** [comment]
Closed by f1676929e83da59f2340e6803f7c5fdbf801b220

**Comment by amoeba** [comment]
Closed by f1676929e83da59f2340e6803f7c5fdbf801b220 cf59fde7b54a2e537adad9695f04aeed1712e02d
